guides/assets/stylesrc/_dark.scss

```diff
@@ -101,9 +105,9 @@
       color: #fff;
     }
 
-    nav#topNav {    background-color: $gray-900;
+    #mobile-navigation-bar {    background-color: $gray-900;
```

Comment:
```suggestion
    #mobile-navigation-bar {
      background-color: $gray-900;
```

Reply:
(something something we should use an auto-formatter imho hehe)

Reply:
Yes :)